### ISSUE TITLE
failed to find any PEM data in certificate input

### ISSUE THREAD

**Issue by UtsavChokshiCNU** [liangrog/admission-webhook-server#5]
`admission-webhook-server` is failing with following error : 
```
2020/08/20 06:59:07 Registering handlers...                                                                                                                                       
2020/08/20 06:59:07 PodNodesSelector registered using path /mutate/pod-nodes-selector                                                                                             
2020/08/20 06:59:07 Starting admission webhook server...                                                                                                                          
2020/08/20 06:59:07 tls: failed to find any PEM data in certificate input
```

List of files generated after running `ssl.sh` script :
> sh ssl.sh admission-webhook.tools.svc
```
ca.crt
ca.key
ca.srl
server.pem
tls.crt
tls.key
```

Namespace : `tools` already exist.

Following is helm installation command I used :
> helm install . --name  admission-webhook-server --namespace tools

**Comment by shubhamitc** [comment]
copy the ssl folder from charts helm to charts folder. this should work after that.

**Comment by liangrog** [comment]
Sorry guys, overlooked this bug. Fixed in v1.0.1
